packages/react-native-session-replay/src/SessionReplay.ts

```diff
@@ -12,6 +12,49 @@ export enum SessionReplayPrivacy {
     MASK_USER_INPUT = 'MASK_USER_INPUT'
 }
 
+export enum ImagePrivacyLevel {
+    /**
+     * Only images that are bundled within the application will be recorded.
+     *
+     * On Android, all images larger than 100x100 dp will be masked, as we consider them non-bundled images.
```

Comment:
no need to add the note about iOS?

Reply:
Maybe I can rephrase it. The idea is that iOS knows which images are bundled and which are not, while Android has its own logic to determine if an image is "bundled" or not, based on its size, and I'd like to make it clear in the comment. What do you think?

Reply:
okay, makes sense, we can keep it like that then.